Source/WTF/wtf/WeakPtr.h

```diff
@@ -123,7 +123,7 @@ template<typename T, typename WeakPtrImpl> class WeakPtr {
     T* get() const
     {
         // FIXME: Our GC threads currently need to get opaque pointers from WeakPtrs and have to be special-cased.
```

Comment:
Should move this comment into your new function.

Reply:
Done